src/client/dfuse/dfuse_fuseops.c

```diff
@@ -81,6 +81,13 @@ dfuse_fuse_init(void *arg, struct fuse_conn_info *conn)
 	conn->max_read = fs_handle->dpi_max_read;
 	conn->max_write = fs_handle->dpi_max_read;
 
+	if (fs_handle->dpi_info->di_caching) {
+		conn->max_readahead = (1024 * 1024);
```

Comment:
hm, shouldn't we disable kernel readahead in this case since we do it?

Reply:
Ok, I can do that.